Documentation/mkfs.btrfs.rst

```diff
@@ -160,6 +160,10 @@ OPTIONS
         directory.  The option *--rootdir* must also be specified, and *subdir* must be an
         existing subdirectory within it.  This option can be specified multiple times.
 
+--default-subvol <subdir>
+        As *--subvol*, except that it also sets the subvolume as the default for the
+        filesystem.  This option can only be specified once.
```

Comment:
Just a comment, this semantic sounds ok. Alternatively it could be --subvol DIR and a separate --default-subvol DIR that will only set the default bit, but it's more convenient to do it in one go.

Reply:
Yes, somebody else also suggested that, but I couldn't really see what the benefit was.